app/helpers/view_components/profile/header.rb

```diff
@@ -23,6 +24,14 @@ def top_three_tracks
         ::Track.where(id: track_ids).
           order(Arel.sql("FIND_IN_SET(id, '#{track_ids.join(',')}')"))
       end
+
+      def header_tags
+        tags = []
+        tags << { class: "tag staff", icon: :logo, title: "Exercism Staff" } if @user.staff?
+        tags << { class: "tag maintainer", icon: :maintaining, title: "Maintainer" } if @user.maintainer?
+        tags << { class: "tag insider", icon: :insiders, title: "Insider" } if @user.insider?
```

Comment:
@iHiD Is this the right order? As in: do maintainer and staff tags have precedence over insider?

Reply:
Do we want to feature flag this?

Reply:
> @iHiD Is this the right order? As in: do maintainer and staff tags have precedence over insider?

Yes

> Do we want to feature flag this?

Yes for today.

Reply:
Okay, I added the feature flag